demo/admin/src/App.tsx

```diff
@@ -7,6 +7,7 @@ import "typeface-open-sans";
 
 import { ApolloProvider } from "@apollo/client";
 import { ErrorDialogHandler, MasterLayout, MuiThemeProvider, RouterBrowserRouter, SnackbarProvider } from "@comet/admin";
+import { MlServiceContext } from "@comet/admin/lib/ml/MlServiceContext";
```

Comment:
Please don't import anything that's not part of the public API (i.e. no /lib/... imports)